deepchecks/vision/metrics_utils/detection_precision_recall.py

```diff
@@ -79,7 +79,10 @@ def compute(self):
                 **self._compute_ap_recall(ev["scores"], ev["matched"], ev["NP"])
             }
         if self.return_ap_only:
-            res = torch.tensor([res[k]["AP"] for k in sorted(res.keys())])
+            res = torch.tensor([
```

Comment:
I think that's OK. This may happen in the case the model detected a class that doesn't exist in the test set.
Anyway, we are planning to replace this module in the near future.